### PR TITLE
Update release notes for milestone 2

### PR DESCRIPTION
**Pull request by SueChaplain**
Updated to include HotSpot compatibility options, JITServer
tech preview and the removal of a Javadoc issue.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

File: doc/release-notes/0.18/0.18.md [hunk 1]
```diff
@@ -142,13 +153,6 @@ The v0.18 release contains the following known issues and limitations:
...
 </tr>
 
-<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/897">#897</a></td>
-<td valign="top">Javadoc not building some OpenJ9 extensions.</td>
-<td valign="top">All platforms (OpenJDK 8 only)</td>
-<td valign="top">Some API documentation is missing in the <code>build/&lt;platform&gt;/docs</code> directory.</td>
-<td valign="top">None</td>
-</tr>
-
 <tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/898">#898</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: This one can also be removed.

**Comment by mpirvu** [comment]
The copyright needs to be advanced to 2020

**Comment by DanHeidinga** [comment]
There's some inconsistent numbering for the release in this document - sometimes it says `0.18` and other times `0.18.0`.  Can `0.18.0` be used consistently thruoghout?

File: doc/release-notes/0.18/0.18.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!--
-* Copyright (c) 2019, 2019 IBM Corp. and others
+* Copyright (c) 2020, 2020 IBM Corp. and others
```
Comment (mstoodle) [inline doc/release-notes/0.18/0.18.md]: shouldn't this be 2019, 2020 ?

Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: Yes probably, as we had something published for MS1. I was fixated on it being a January release. Will change.

File: doc/release-notes/0.18/0.18.md [hunk 3]
```diff
@@ -142,24 +153,10 @@ The v0.18 release contains the following known issues and limitations:
...
-</tr>
-
 </tbody>
 </table>
 
 
 ## Other changes
 
-A full commit history for this release is available at [Eclipse OpenJ9 v0.18.0](https://github.com/eclipse/openj9/releases/tag/openj9-0.18.0).
+A full commit history for this release is available at [Eclipse OpenJ9 v0.18.0.0](https://github.com/eclipse/openj9/releases/tag/openj9-0.18.0).
```
Comment (pshipton) [inline doc/release-notes/0.18/0.18.md]: There is an extra `.0` here now.

Comment (SueChaplain) [reply doc/release-notes/0.18/0.18.md]: Urgh ... the hazards of global search and replace :-/ Fixed

**Pull request merged**